packages/client/components/bottom-drawer/index.tsx

```diff
@@ -0,0 +1,98 @@
+import React, { useState } from 'react'
+import { Grid } from '@material-ui/core'
+import styled from 'styled-components'
+import { useSwipeable } from 'react-swipeable'
+
+const BottomDrawerContainer = styled(Grid)`
+  height: ${props => props.drawerHeight};
```

Comment:
Are these props inferred? If not, it might be better to use styled-component's `withProps` here.

Reply:
> Are these props inferred? If not, it might be better to use styled-component's `withProps` here.

What is `withProps`? I typed the props. Is it the same thing?

Reply:
How does typescript know the props of this component, `BottomDrawerContainer`?

Reply:
I made it like this `height: ${(props: { drawerHeight: string }) => props.drawerHeight};` Github seems showing the older version.